### ISSUE TITLE
Multiline literal types in CRLF ending files are not normalized to LF

### ISSUE THREAD

**Issue by jairinhohw** [microsoft/pyright#9681]
Comparing a type of a multiline literal in a file with CRLF line ending does not match the same literal in a file with LF ending.

This makes the code below type check if the file has CF line ending, but error if the file has a CRLF line ending.

```python
from typing import Literal

a: Literal[
    "a\nb"
] = """a
b"""

print(repr(a))
```

running the file prints `'a\nb'` in both cases, so the line ending of the python file should not matter.

**VS Code extension or command-line**
Tested with both the pylance extension and the pyright command line tool version `1.1.391`

Error returned, with CRLF file:
```
/tmp/tst/tst.py
  /tmp/tst/tst.py:5:5 - error: Type "Literal['a\r\nb']" is not assignable to declared type "Literal['a\nb']"
    "Literal['a\r\nb']" is not assignable to type "Literal['a\nb']" (reportAssignmentType)
```

**Comment by erictraut** [comment]
Thanks for the bug report. I couldn't find any official documentation that indicates this is the behavior of the Python runtime, but I confirmed the behavior through experimentation. I've updated pyright's logic accordingly. This will be addressed in the next release.

**Comment by erictraut** [comment]
This is included in pyright 1.1.392.
